src/skeletons.ts

```diff
@@ -337,6 +338,44 @@ export const BORDER_SKELETONS: Record<
       right: '',
     },
   },
+  'vertical-rows-with-outline': {
+    data: {
+      cross: '│',
+      left: '│',
+      line: ' ',
+      right: '│',
+    },
+    footer: {
+      cross: '┴',
+      left: '└',
+      line: '─',
+      right: '┘',
+    },
+    header: {
+      cross: '─',
+      left: '┌',
+      line: '─',
+      right: '┐',
+    },
+    headerFooter: {
+      cross: '┬',
+      left: '├',
+      line: '─',
+      right: '┤',
+    },
+    heading: {
+      cross: ' ',
+      left: '│',
+      line: ' ',
+      right: '│',
+    },
+    separator: {
+      cross: '',
+      left: '',
+      line: '',
+      right: '',
+    },
+  },
```

Comment:
Honestly don't care if you want to rename or delete this new style. Here is the difference:
<img width="350" alt="Screenshot 2024-10-03 at 12 38 16 PM" src="https://github.com/user-attachments/assets/0870ecdb-aac2-4cbd-b359-6a04d997f228">